tools/hornet-flow/src/hornet_flow/models.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=missing-class-docstring
+# pylint: disable=missing-function-docstring
+
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Annotated, List, Literal, Union
+
+from pydantic import AnyUrl, BaseModel, ConfigDict, Field, HttpUrl, RootModel
```

Comment:
sim4life is still on pydantic v1 so this package will not install into the s4l venv. Not sure if this is a problem though

Reply:
i have noticed a moment ago ... yes, it is a problem. I will downgrade